src/pages/CustomerRequisitionPage.js

```diff
@@ -168,18 +168,18 @@ export const CustomerRequisitionPage = ({ requisition, runWithLoadingIndicator,
   }, []);
 
   const {
-    newPageTopSectionContainer,
-    newPageTopLeftSectionContainer,
-    newPageTopRightSectionContainer,
-  } = newPageStyles;
+    pageTopSectionContainer,
+    pageTopLeftSectionContainer,
+    pageTopRightSectionContainer,
+  } = globalStyles;
   return (
     <DataTablePageView>
-      <View style={newPageTopSectionContainer}>
-        <View style={newPageTopLeftSectionContainer}>
+      <View style={pageTopSectionContainer}>
+        <View style={pageTopLeftSectionContainer}>
           {renderPageInfo()}
```

Comment:
Nit:

Returning method calls inside JSX looks a bit weird to me. Might be nicer to assign it to a component variable:

```
const PageInfo = renderPageInfo()
return (
  <DataTablePageView> 
  ...
   <View style=PageTopLeftSectionContainer>
     <PageInfo />
   </View>
  ...
...
```
```

Reply:
Unfortuantely creating components like this will actually start to re-mount components, rather than re-render them. Sucks cause it is way nicer

Reply:
> Unfortuantely creating components like this will actually start to re-mount components, rather than re-render them. Sucks cause it is way nicer

Even if you memoise?